packages/bpk-component-chip/examples.js

```diff
@@ -151,6 +151,74 @@ class StatefulDismissibleChipsExample extends React.Component<
   }
 }
 
+class StatefulRadioGroupChipsExample extends React.Component<
+  { type: $Keys<typeof CHIP_TYPES> },
+  { chips: Array<string>, updates: Array<string>, selectedIndex: number },
+> {
+  constructor() {
+    super();
+    this.state = {
+      selectedIndex: 0,
+      chips: ['Nara', 'Monteverde', 'Panjin', 'Kolkata'],
+      updates: [],
+    };
+  }
+
+  selectChip = (indexToSelect: number) => {
+    const selectedChip = this.state.chips[indexToSelect];
+
+    this.setState(prevState => {
+      return {
+        selectedIndex: indexToSelect,
+        updates: [...prevState.updates, `${selectedChip} selected.`],
+      };
+    });
+  };
+
+  render() {
+    return (
+      <div>
+        <div role="radiogroup" aria-labelledby="favorite-color-label">
+          <BpkText
+            id="favorite-color-label"
+            aria-hidden
+            tagName="h1"
+            textStyle={TEXT_STYLES.lg}
+            weight={WEIGHT_STYLES.bold}
+          >
+            Favourite colour
+          </BpkText>
+          {this.state.chips.map((chip, index) => (
+            <BpkSelectableChip
+              {...this.props}
+              role="radio"
+              key={index.toString()}
+              onClick={() => {
+                this.selectChip(index);
+              }}
+              selected={this.state.selectedIndex === index}
+              disabled={index === 3}
+              className={getClassName('bpk-chip-examples__chip')}
+            >
+              {chip}
+            </BpkSelectableChip>
+          ))}
+        </div>
+        <AriaLiveDemo
+          className={getClassName('bpk-banner-alert-examples__component')}
+        >
+          {this.state.updates.map(u => (
+            <>
+              {u}
```

Comment:
Single-letter variable names?

![i don't think](https://user-images.githubusercontent.com/73652/123284801-708ff480-d504-11eb-935a-d26a1c024722.gif)

Seriously though, can't hurt to make it more explicit and aid self documentation.